### PR TITLE
update centos systemd documentation for CentOS base containers

### PR DESCRIPTION
**Pull request by jperrin**

### CODE REVIEW

File: centos/content.md [hunk 1]
```diff
@@ -26,7 +26,9 @@ By default, the CentOS containers are built using yum's `nodocs` option, which h
 
 # Systemd integration
 
-Currently, systemd in CentOS 7 has been removed and replaced with a `fakesystemd` package for dependency resolution. This is due to systemd requiring the `CAP_SYS_ADMIN` capability, as well as being able to read the host's cgroups. If you wish to replace the fakesystemd package and use systemd normally, please follow the steps below.
+Systemd is now included in both the centos:7 and centos:latest base
+containers, but it is not active by default. In order to use systemd, you will
+need to include text similar to the example Dockerfile below:
```
Comment (yosifkit) [inline centos/content.md]: Can you make these one line; that is what the Travis build is complaining about: https://travis-ci.org/docker-library/docs/builds/100199255#L155-L165.

File: centos/content.md [hunk 2]
```diff
@@ -34,8 +36,6 @@ Currently, systemd in CentOS 7 has been removed and replaced with a `fakesystemd
 FROM centos:7
 MAINTAINER "you" <your@email.here>
 ENV container docker
-RUN yum -y swap -- remove fakesystemd -- install systemd systemd-libs
-RUN yum -y update; yum clean all; \
 (cd /lib/systemd/system/sysinit.target.wants/; for i in *; do [ $i ==
```
Comment (yosifkit) [inline centos/content.md]: I think this line needs a `RUN` now :wink:.

**Comment by yosifkit** [comment]
Also, feel free to drop the changes to `README.md` if you don't want to keep running `update.sh`; the bot will periodically regenerate it and also update the Docker Hub after a merge.

**Comment by jperrin** [comment]
Ah crap. you're right on both. will fix.

**Comment by yosifkit** [comment]
LGTM

File: centos/content.md [hunk 3]
```diff
@@ -74,10 +72,10 @@ $ docker build --rm -t local/c7-systemd-httpd
 
 ## Running a systemd enabled app container
 
-In order to run a container with systemd, you will need to use the `--privileged` option mentioned earlier, as well as mounting the cgroups volumes from the host. Below is an example command that will run the systemd enabled httpd container created earlier.
+In order to run a container with systemd, you will need to mount the cgroups volumes from the host. Below is an example command that will run the systemd enabled httpd container created earlier.
 
 ```console
-$ docker run --privileged -ti -v /sys/fs/cgroup:/sys/fs/cgroup:ro -p 80:80 local/c7-systemd-httpd
+$ docker run -ti -v /sys/fs/cgroup:/sys/fs/cgroup:ro -p 80:80 local/c7-systemd-httpd
```
Comment (tianon) [inline centos/content.md]: Oooh, systemd without `--privileged`!  Nice work! :+1:

**Comment by tianon** [comment]
LGTM

File: centos/content.md (hunk 3 above)
Comment (jperrin) [inline centos/content.md]: it may still need privileged for some operations, but the base container itself runs fine without.

**Pull request merged**